components/helm-broker/docs/controller-logic.md

```diff
@@ -0,0 +1,52 @@
+## Controller logic
```

Comment:
this doc describes how controller logic should be implemented (created for developer who will implement that).

we have to rewrite the content so that it is also helpful to the end-user. End-user should know how the Helm Broker will process created Configuration (adding, updating and deleting them)